enterprise/frontend/src/embedding-sdk/cli/snippets/nextjs-page-snippet.ts

```diff
@@ -0,0 +1,14 @@
+// Next.js page for /app/analytics-demo/page.tsx or /pages/analytics-demo.tsx
+export const getNextJsAnalyticsPageSnippet = ({
```

Comment:
this function seems to be called from both pages and app router. It shouldn't be in `nextjs-page-snippet` anymore. You could just combine `nextjs-page-snippet` and `nextjs-app-snippets` tbh and name the combined file something more generic like `nextjs-snippets`

Also, I just noted that this name doesn't end with `s` like `nextjs-app-snippets`